deps/postgres/ca.crt

```diff
@@ -0,0 +1,17 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
You are adding certificates twice, one under `deps/config/certificates/....` and here, I would rather keep this ones though

Reply:
discussed. We can reuse the same client certs as http client